charter/index.html

```diff
@@ -269,9 +269,12 @@ <h3>
         <section id="dependencies">
           <h3>Dependencies</h3>
 
-          <p>Depending on the W3C Solid Community Group progress, including consideration for <a href="https://www.w3.org/2021/Process-20211102/#adequate-implementation">adequate implementation experience</a>, the Group will also produce W3C Recommendations for the following documents:</p>
+          <p>Depending on the Working Group progress, including consideration for <a href="https://www.w3.org/2021/Process-20211102/#adequate-implementation">adequate implementation experience</a>, the Group will also produce W3C Recommendations for the following documents:</p>
 
           <dl>
+              <dt><a href="https://www.w3.org/2005/Incubator/webid/spec/identity/">WebID</a></dt>
+              <dd>A simple universal identification mechanism that is distributed, openly extensible, improves privacy, security and control over how each person can identify themselves in order to allow fine grained access control to their information on the Web.</dd>
+
```

Comment:
should we add a link to the WebID github repo now? I don't think the current spec has a link to that.

Reply:
https://github.com/solid/solid-wg-charter/pull/40/commits/7d6fbf16f51744d8650efe5ba6a1fd640ace580c okay?